pkg/kubelet/util/manager/manager.go

```diff
@@ -49,12 +50,12 @@ type Store interface {
 	// AddReference adds a reference to the object to the store.
 	// Note that multiple additions to the store has to be allowed
 	// in the implementations and effectively treated as refcounted.
-	AddReference(namespace, name string)
+	AddReference(namespace, name string, podUID types.UID)
```

Comment:
Can we make it pod-agnostic?

We don't rely anywhere on the fact that it's actually a pod here. All we want is that it's a UID of the object that is referencing our object.

Can we change the name of it (here and everywhere else) to avoid mentioning pod? E.g. `referencingObjUID` or sth like that?

Reply:
This UID should not appear to be UID of namespace/name.

How about `referencedUID` or `refUID`?

Reply:
maybe something like `referencedFrom types.UID` would be clearer

Reply:
I like that. @ruquanzhao - can you please update accordingly?

Reply:
Done.